recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -366,7 +369,69 @@
    "id": "0f8cd0d8",
    "metadata": {},
    "source": [
-    "There are multiple parameters such as the name of the source, the interval of validity of the model (START to FINISH), the frequencies of rotation and its derivatives (F0,F1,F2). There are other additional parameters that can be checked in the [PINT documentation](https://nanograv-pint.readthedocs.io)"
+    "There are multiple parameters such as the name of the source, the frequencies of rotation and its derivatives (F0,F1,F2), the dispersion measure, etc. Check the [PINT documentation](https://nanograv-pint.readthedocs.io) for a list of additional parameters. To obtain the complete set of parameter from the ephemeris file, one can simply print the model:\n",
+    "`print(model)`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f02f0ca5-09f2-44d4-ad76-518f8964b6a1",
+   "metadata": {},
+   "source": [
+    "As mentioned previously, we should ensure the time of the observation lies within the ephemeris time definition. In our example, we only have one run, so we can check that manually:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2584b571-93a9-4fb6-b8e2-60b8810190f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Ephemeris time definition:\\n{model.START.value} - {model.FINISH.value}\")\n",
+    "print(f\"Observation time definition:\\n{observation.tstart} - {observation.tstop}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86041174-5ec4-43ba-9035-cf567b47a3bb",
+   "metadata": {},
+   "source": [
+    "If you have several observations that they are sorted by time, you can manually check for the start time of the first observation and the stop time of the last one. Otherwise, you can create a small function like the following one:"
```

Comment:
```suggestion
    "If you have several observations that are sorted by time, you can manually check for the start time of the first observation and the stop time of the last one. Otherwise, you can create a small function like the following one:"
```